.github/workflows/CICD.yml

```diff
@@ -624,6 +613,13 @@ jobs:
         case '${{ matrix.job.os }}' in
           macos-latest) brew install coreutils ;; # needed for testing
         esac
+        case '${{ matrix.job.os }}' in
+          *linux*)
+            sudo sed -i 's/:,/:runner name,/' /etc/passwd  # add a full name each account with a gecos field but no full name
+            FAKE_UTMP='[7] [999999] [tty2] [runner] [tty2] [] [0.0.0.0] [2022-02-22T22:22:22,222222+00:00]'
+            (utmpdump /var/run/utmp ; echo $FAKE_UTMP) | sudo utmpdump -r -o /var/run/utmp  # create a fake login entry for the GH runner
+            ;; # helps pinky tests
+        esac
```

Comment:
I'm getting some understanding from comments and variable naming, but this is a little opaque to me.
Could you document a little more "why" for this block (locally), expanding on "helps pinky tests"? Maybe just a couple of explanatory lines?
I think it'll help future me (and maybe future you) understand why it helps and if it can be and how it should be modified in the future.

Thanks.

Reply:
I've updated the comment, how does it look now?